src/js/components/DeploymentsListComponent.jsx

```diff
@@ -107,6 +168,11 @@ var DeploymentListComponent = React.createClass({
               Error fetching deployments. Refresh to try again.
             </td>
           </tr>
+          <tr className={errorMessageClassSet}>
+            <td className="text-center text-danger" colSpan="5">
+              {state.errorMessage}
+            </td>
+          </tr>
```

Comment:
I'm suggesting switching the order of the messages, but keep them both above apps/deployments list